cli/internal/cmd/apply.go

```diff
@@ -293,6 +297,23 @@ type applyCmd struct {
 
 type applier interface {
```

Comment:
nit: this name doesn't fit well. How about `initer`?

Reply:
The goal is to have this interface implement all methods for clients that will be shared by the CLI and the Terraform provider, so `initer` does not fit that description.
See [AB#3557](https://dev.azure.com/Edgeless/Edgeless/_workitems/edit/3557) for more details.